<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoContent</class>
 <widget class="QWidget" name="VideoContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conteúdo em Vídeo</string>
  </property>
  <widget class="QTabWidget" name="tabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>721</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_content">
    <attribute name="title">
     <string>Conteúdo</string>
    </attribute>
    <widget class="QPlainTextEdit" name="txt_media_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>41</height>
      </rect>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_choose_video">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Escolher Vídeo</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/i/folder_open.png</normaloff>:/i/folder_open.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QWidget" name="player_holder" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>721</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_position">
    <attribute name="title">
     <string>Posição do Vídeo</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_behavior">
    <attribute name="title">
     <string>Comportamento do Vídeo</string>
    </attribute>
    <widget class="QWidget" name="volume_control_widget" native="true">
     <property name="geometry">
      <rect>
       <x>721</x>
       <y>20</y>
       <width>301</width>
       <height>237</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_begin_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>16</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tempo de Início:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_begin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_duration">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>16</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Duração (segundos):</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_end_time">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>16</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tempo de Término:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_end">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_duration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>590</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/check.png</normaloff>:/i/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>590</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/no.png</normaloff>:/i/no.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
